Sources/SecureXPC/Server/XPCServiceServer.swift

```diff
@@ -55,4 +66,23 @@ internal class XPCServiceServer: XPCServer {
 		// XPC services are application-scoped, so we're assuming they're inheritently safe
 		true
 	}
+
+    public override var serviceName: String? {
+        xpcServiceName
+    }
+
+    public override var endpoint: XPCServerEndpoint {
+        guard let connection = self.connection else {
+            fatalError("""
+                An XPCServer for an XPC service can only have its `connection` exported to an `endpoint` after
+                a `connection` has been created by calling `start()`.
```

Comment:
I think this can have the same message used for `XPCMachService`, something like:
> An XPCServer's endpoint can only be retrieved after start() has been called on it.

Reply:
Deduping against https://github.com/trilemma-dev/SecureXPC/pull/23#discussion_r762503640